test/test.js

```diff
@@ -578,7 +579,7 @@ describe('parser (exif data)', () => {
 
 	describe('cameras', () => {
 
-		const options = { xmp: true, flir: true }
+		const options = { xmp: true, flir: true, sof: true }
```

Comment:
If the sof option is set to true but the exif data does not contain a sof segment, will it just be undefined?

Reply:
Yes indeed :). 
![image](https://user-images.githubusercontent.com/17416198/95055499-a29dc400-06f3-11eb-8c58-990f56112290.png)